sources/packages/backend/libs/integrations/src/sfas-integration/sfas-integration.models.ts

```diff
@@ -17,4 +17,5 @@ export class DownloadResult {
 export class ProcessSftpResponseResult {
   summary: string[] = [];
   success: boolean;
+  error: string[] = [];
```

Comment:
I would refrain from improving this interface since we have another process summary that would replace this one.